src/asyncrpcoperation.cpp

```diff
@@ -44,6 +48,7 @@ AsyncRPCOperation::AsyncRPCOperation(const AsyncRPCOperation& o) :
 }
 
 AsyncRPCOperation& AsyncRPCOperation::operator=( const AsyncRPCOperation& other ) {
```

Comment:
this doesn't really feel like something that should be copy assignable - i.e. what happens if i take a copy and cancel the copy? 

it should be  move assignable - but i don't see any reason why the default move assignment wouldn't work. if you just delete this operator and replace any assignments with a move?

i.e. `AsyncRPCOperation x = y;` becomes `AsyncRPCOperation x = std::move(y)`

Reply:
Good insight; agreed

Reply:
Ref https://github.com/BTCPrivate/BTCP-Rebase/pull/33